### PR TITLE
Modify the default project namespace

### PR DESCRIPTION
**Pull request by jcantrill**
* Modify the default project namespace to openshift-logging

Ref: https://bugzilla.redhat.com/show_bug.cgi?id=1535300

### CODE REVIEW

**Comment by jcantrill** [comment]
/test json-file
/test/logging

File: test/multi_tenancy.sh [hunk 1]
```diff
@@ -154,11 +156,11 @@ done
 # use different usernames, otherwise you'll get this odd error:
 # # oc login --username=loguser --password=loguser
 # error: The server was unable to respond - verify you have provided the correct host and port and that the server is currently running.
-LOG_NORMAL_USER=${LOG_NORMAL_USER:-loguser}
-LOG_NORMAL_PW=${LOG_NORMAL_PW:-loguser}
+LOG_NORMAL_USER=${LOG_NORMAL_USER:-loguser-$RANDOM}
```
Comment (ewolinetz) [inline test/multi_tenancy.sh]: where is `$RANDOM` coming from?

Comment (jcantrill) [reply test/multi_tenancy.sh]: std bash function

**Comment by richm** [comment]
I must admit I'm a bit confused as to how this is supposed to work.
There are many places in the tests where we are checking for an index like `/project.openshift-logging.*`.  But according to fluentd, _there should never be such an index_ - that is, because of https://github.com/openshift/origin-aggregated-logging/blob/master/fluentd/run.sh#L4 - _all records from namespace openshift-logging should go into .operations_.

So, how is it that the tests are working?

**Pull request merged**